lib/fork.js

```diff
@@ -56,20 +45,36 @@ module.exports = function (args) {
 						type: 'test'
 					});
 				}
+
 				resolve(testResults);
 			} else {
-				reject(new Error('Never got test results from: ' + file));
+				reject(new Error('Test results were not received from: ' + file));
 			}
 		});
 	});
 
 	// emit `test` and `stats` events
 	ps.on('message', function (event) {
+		if (!event.ava) {
+			return;
+		}
+
+		event.name = event.name.replace('ava-', '');
```

Comment:
`'ava-'` → `/^ava-/` just to be sure we don't strip any future message with `ava-` in the middle. I know, edge case, but this gives clearer intent too.